queries.yaml

```diff
@@ -1740,6 +1741,58 @@ post_process:
       source_layer: buildings
       target_value_type: int
 
+  - fn: vectordatasource.transform.drop_properties
+    params:
+      source_layer: buildings
+      start_zoom: 0
+      end_zoom: 17
+      properties:
+        - landuse_kind
+      where: >-
+        landuse_kind not in ['aerodrome', 'airfield', 'cemetery', 'college', 'conservation', 'forest', 'garden', 'golf_course', 'hospital', 'national_park', 'nature_reserve', 'park', 'protected_area', 'runway', 'school', 'taxiway', 'university']
+
+  - fn: vectordatasource.transform.drop_properties
+    params:
+      source_layer: buildings
+      start_zoom: 0
+      end_zoom: 16
+      properties:
+        - kind_detail
+      where: >-
+        kind_detail not in ['airport', 'civic', 'church', 'college', 'commercial', 'fire_station', 'industrial', 'military', 'hospital', 'office', 'public', 'prison', 'retail', 'school', 'sports_centre', 'stadium', 'station', 'synagogue', 'train_station', 'university', 'warehouse']
+
+  - fn: vectordatasource.transform.drop_properties
+    params:
+      source_layer: buildings
+      start_zoom: 0
+      end_zoom: 16
+      properties:
+        - building_material
+        - layer
+        - location
+        - osm_relation
+        - roof_color
+        - roof_height
+        - roof_material
+        - roof_orientation
+        - roof_shape
+        - root_id
+        - volume
+
+  - fn: vectordatasource.transform.drop_properties
+    params:
+      source_layer: buildings
+      start_zoom: 0
+      end_zoom: 17
```

Comment:
This is removing names and addr properties from a 512px sized zoom 16 tile. They should only be removed from a zoom 15 tile.

Reply:
Example OSM feature: https://www.openstreetmap.org/way/108160358

Reply:
made a change on tilequeue let's see whether that can fix it: tilezen/tilequeue#407

Reply:
Confirmed Tilequeue changes result in expected features being keep at zoom 16 in 512px space and dropped at zoom 15.